code/modules/paperwork/photocopier.dm

```diff
@@ -332,7 +333,7 @@
 		copyitem = null
 	updateUsrDialog()
 
-/obj/machinery/photocopier/proc/check_ass() //I'm not sure wether I made this proc because it's good form or because of the name.
+/obj/machinery/photocopier/proc/check_ass() //I'm not sure weither I made this proc because it's good form or because of the name.
```

Comment:
```suggestion
/obj/machinery/photocopier/proc/check_ass() //I'm not sure whether I made this proc because it's good form or because of the name.
```
neither the original or your change are right here, I'm afraid!

Reply:
I may just leave it in because it reflects the quality of photopcopier.dm